src/index.jsx

```diff
@@ -21,6 +21,10 @@ if (
   })
 }
 
+if (process.env.REACT_APP_ENV === "test") {
+  import("./mocks/browser").then(({ worker }) => worker.start())
```

Comment:
when do _not_ want msws to run ah?

Reply:
we only want msw to run for storybook. shouldn't be run in other environments because it will mock endpoints, returning false data

Reply:
oo right sorry I thought this was a `.story` file